service/src/main.rs

```diff
@@ -697,25 +703,6 @@ fn execute_trusted_calls<E: Sidechain>(enclave_api: &E) {
 	};
 }
 
-fn init_shard(shard: &ShardIdentifier) {
-	let path = format!("{}/{}", SHARDS_PATH, shard.encode().to_base58());
-	println!("initializing shard at {}", path);
-	fs::create_dir_all(path.clone()).expect("could not create dir");
-
-	let path = format!("{}/{}", path, ENCRYPTED_STATE_FILE);
-	if Path::new(&path).exists() {
-		println!("shard state exists. Overwrite? [y/N]");
-		let buffer = &mut String::new();
-		stdin().read_line(buffer).unwrap();
-		match buffer.trim() {
-			"y" | "Y" => (),
-			_ => return,
-		}
-	}
-	let mut file = fs::File::create(path).unwrap();
-	file.write_all(b"").unwrap();
-}
-
```

Comment:
This is removed in favor of an e-call that uses the state handler to initialize a shard.